### ISSUE TITLE
deno bundle results in `Module not found` for `util-buffer-from`

### ISSUE THREAD

**Issue by kevin-mitchell** [christophgysin/aws-sdk-js-v3#36]
## Describe the bug

As a new Deno developer looking to use the AWS SDK, when running `deno bundle` on a project that has `import * as awsSdk from "https://deno.land/x/aws_sdk@v3.32.0-1/client-secrets-manager/mod.ts";` I'm seeing an error `Module not found "https://jspm.dev/npm:@aws-sdk/util-buffer-from@3.183.0"`:

## Your environment

MacBook Pro M1 Pro, MacOS Monterey

#### SDK version number
`[@aws-sdk/package-name@version](https://deno.land/x/aws_sdk@v3.32.0-1/client-secrets-manager/mod.ts)`

## Details of the deno version

```
deno 1.26.0 (release, aarch64-apple-darwin)
v8 10.7.193.3
typescript 4.8.3
```

## Steps to reproduce
Please share code or minimal repo, and steps to reproduce the behavior.

#### Observed behavior

(_note I'm using unstable and no-check here because they were mentioned in one of the closed issues that seem related, but I get the same error regardless of whether I provide these flags or not_)

```
deno bundle --unstable --no-check src/deno/get-content.ts > test.ts
Download https://jspm.dev/npm:@aws-sdk/util-buffer-from@3.183.0
error: Module not found "https://jspm.dev/npm:@aws-sdk/util-buffer-from@3.183.0".
    at https://jspm.dev/@aws-sdk/hash-node:1:8
```

#### Expected behavior

I had hoped this would result in a successful bundle of my application.

## Additional context

_I did look through previously closed issues and found [this](https://github.com/christophgysin/aws-sdk-js-v3/issues/32) and [this](https://github.com/christophgysin/aws-sdk-js-v3/issues/29), both of which of course seem very related but I wasn't able to understand the steps required to resolve the issue_.

_I'm not actually sure if I'm correctly including this library_ - I am brand new to Deno, and have cobbled together what seems to be a reasonable example from what I've seen on the internet (reddit, stackexchange, etc). 

In my specific case I'm interested in using the `client-secrets-manager`, e.g. 

```typescript
import * as awsSdk from "https://deno.land/x/aws_sdk@v3.32.0-1/client-secrets-manager/mod.ts";

...

const secretManager = new awsSDK.SecretsManager({});

...
```

If there is a better way of doing this, or if I'm incorrectly including the SDK, I'd love to know!

**Comment by jasonsoutar** [comment]
I followed the example in [the issue you mentioned](https://github.com/christophgysin/aws-sdk-js-v3/issues/32)
```
$ curl https://api.jspm.io/buildil-buffer-from@3.183.0
{"status":"Build request sent to queue."}
```
It took about 30 minutes but now that package seems to be available at `https://jspm.dev/npm:@aws-sdk/util-buffer-from@3.183.0`

**Comment by kevin-mitchell** [comment]
Thank you so much @jasonsoutar for taking the time to look at that, and I'm sorry it took your comment for me to realize that this was something I could do myself. I had found the issue (obviously I suppose, given I linked to it!) but in my head I assumed that this command was something that could only be run by an admin, or something along those lines. I was not (and am not!) familiar with jspm.io to be honest, but still I coudl have at least TRIED to run the command you referenced.

Anyway, thank you very much for your time. I'm going to close this ticket.

I will add that I'm wondering at this point if an alternate way of including the AWS SDK as I need is to use the `npm import` option (see: https://deno.com/blog/changes).

Anyway, thanks again!
